src/test/java/kz/andersen/java_intensive_13/console/UserConsoleTest.java

```diff
@@ -16,6 +17,7 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
+@Disabled
```

Comment:
and this is disabled?

Reply:
tests are disabled because the UserConsole class no longer in use

Reply:
Can I delete UserConsole class in the project, and tests too

Reply:
You van leave. Just add comment why tests are disabled)